src/Orleans/Core/IStatefulGrain.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Orleans.Core;
+
+namespace Orleans
+{
+    internal interface IStatefulGrain
```

Comment:
why do we need this new interface? Can't Grain<T> have those? 
Or generally, can this extra interface be avoided?